src/ring_theory/fractional_ideal.lean

```diff
@@ -365,6 +367,10 @@ lemma sup_eq_add (I J : fractional_ideal S P) : I ⊔ J = I + J := rfl
 @[simp, norm_cast]
 lemma coe_add (I J : fractional_ideal S P) : (↑(I + J) : submodule R P) = I + J := rfl
 
+@[simp, norm_cast]
+lemma coe_ideal_sup (I J : ideal R) : (↑(I ⊔ J) : fractional_ideal S P) = I + J :=
```

Comment:
Ah, I've realized my confusion - it would have been avoided if you instead wrote this
```suggestion
lemma coe_ideal_sup (I J : ideal R) : ↑(I ⊔ J) = (I + J : fractional_ideal S P) :=
```